### ISSUE TITLE
discussion: overwrite enr options when enr option does not exist on init

### ISSUE THREAD

**Issue by 3xtr4t3rr3str14l** [ChainSafe/lodestar#1748]
i noticed that if i try to do `./bin/lodestar init --enr.ip 9.9.9.9` and i check the enr at enr-viewer.com, it does not show the ip.  should we allow the user to be able to make new enr entries via the enr options overwrites whenever those entries aren't already in the enr?  seems like it could be useful, but i'm also not sure what the original rationale was for not allowing this, so i'm open to hearing about that as well.

**Comment by 3xtr4t3rr3str14l** [comment]
oh wait, nevermind, this does work when you pass it into `beacon` instead of `init`.  is this something we should also allow in `init`?  currently, if you do `./bin/lodestar init --help` the enr overwrites show up as cli options.  either we should remove these enr overwrites for init or add support for them, i suppose.

**Comment by 3xtr4t3rr3str14l** [comment]
this might also be part of a larger discussion about whether or not we want to allow the user to overwrite an *existing* beacon config (i.e. one that has already been initialized).  when i pass any other cli option to `init` when we already have an existing testnet folder and beacon config, it doesn't overwrite the entry.  maybe this is something we need to talk about supporting in general?  they can always update the beacon config file by hand, sure, but seems like it might be potentially convenient to be able to update it via cli regardless of whether there's an existing beacon config or not.

**Comment by dapplion** [comment]
Those are good points, but makes me think: is the init command necessary? Do other clients have a similar command?

**Comment by 3xtr4t3rr3str14l** [comment]
fwiw, it doesn't look like lighthouse does it (source: https://lighthouse-book.sigmaprime.io/testnet-validator.html#step-5-start-lighthouse)
Maybe it's good to keep `init` just if someone wants to have a fresh environment set up that they then want to configure themselves before actually running `beacon`.  If we were to do away with `init`, then useres would have to run the beacon node first if they wanted to tweak the config before running (without actually setting up the .<testnet> folder and beacon.config and all that by hand), which doesn't sound ideal for that scenario.  However, I think that when someone runs `beacon` it shouldn't be required to have to run `init` first.  

tl;dr it seems to me like `init` should be optional, but not taken away.

**Comment by dapplion** [comment]
Closing since init command usage has been deprecated
